CHANGELOG.md

```diff
@@ -22,6 +27,9 @@ This project tries to follow [SemVer 2.0.0](https://semver.org/).
 - Added Swagger spec metadata such as version that equals the version of the
   API, contact information, and license. (#4)
 
+- Changed version of Alpine Docker image used as the base image from 3.13.4
+  -> 3.13.5. (#13)
+
```

Comment:
Looks like a merge conflict resolution error.

This note was removed in favor of the one below.

```suggestion
```